subsys/mgmt/mcumgr/grp/os_mgmt/Kconfig

```diff
@@ -135,6 +135,26 @@ config MCUMGR_GRP_OS_ECHO
 	default y
 	select MCUMGR_SMP_CBOR_MIN_DECODING_LEVEL_2
 
+config MCUMGR_GRP_OS_DATETIME
+	bool "Support for datetime command"
+	depends on RTC
+	select MCUMGR_SMP_CBOR_MIN_DECODING_LEVEL_2
```

Comment:
Adding `depends on $(dt_alias_exists,rtc)` could catch a missing rtc: alias in a nice way :)

Reply:
Nice! Fixed